Sources/Testing/Issues/Issue.swift

```diff
@@ -65,6 +65,7 @@ public struct Issue: Sendable {
 
     /// An issue due to a failure in the underlying system, not due to a failure
     /// within the tests being run.
+    @_spi(ForToolsIntegrationOnly)
```

Comment:
We should leave `.system` as SPI for now since we don't generally want external components to generate system issues. These represent problems with Swift Testing itself. (We could maybe add an additional case e.g. `case integratedToolFailed(description: String)`, what do you think @stmontgomery ?)

Reply:
Just so I understand, you're saying this line is correct? Or is there a more-correct `@_spi` annotation than `ForToolsIntegrationOnly`? [The SPI doc](https://github.com/apple/swift-testing/blob/main/Documentation/SPI.md) only lists `Experimental` and `ForToolsIntegrationOnly`.

Reply:
Yes, adding the SPI marker here is probably the right thing to do, but I'd like @stmontgomery to chime in too.

Reply:
I think it's OK to leave `.system` _without_ an `@_spi` attribute. Although it's true that users _shouldn't_ generally record system issues themselves, I do think it can be useful for them to _match against_ issues of that kind using other APIs like `withKnownIssue`. So personally, I don't think we need to add it as part of this proposal/PR.